api/XMLDocument.json

```diff
@@ -4,12 +4,26 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/XMLDocument",
         "support": {
-          "chrome": {
-            "version_added": "45"
-          },
-          "chrome_android": {
-            "version_added": "45"
-          },
+          "chrome": [
+            {
+              "version_added": "34"
+            },
+            {
+              "version_added": "1",
```

Comment:
The&nbsp;annoying&nbsp;part about&nbsp;that is&nbsp;that, even&nbsp;though the&nbsp;support&nbsp;ranges are&nbsp;contiguous, adding&nbsp;`version_removed` still&nbsp;causes the&nbsp;background to&nbsp;turn from&nbsp;yellow&nbsp;to&nbsp;red.

Also, I&nbsp;can’t&nbsp;set&nbsp;it for&nbsp;`webview_android` due&nbsp;to&nbsp;a&nbsp;bug in&nbsp;the&nbsp;linter.

Reply:
Personally, I think that the red background is applicable to this case, as it indicates “this is old, deprecated behavior that is no longer present in this browser,” and since the clause refers to support under an alias, I’d say that makes sense.

What’s going on with the linter, by the way?

Reply:
It&nbsp;disallows:
```json
"webview_android": {
  "version_added": "≤37",
  "version_removed": "37"
}
```

As&nbsp;it&nbsp;treats it&nbsp;as:
```json
"webview_android": {
  "version_added": "37",
  "version_removed": "37"
}
```

Instead&nbsp;of:
```json
"webview_android": {
  "version_added": true,
  "version_removed": "37"
}
```

When&nbsp;comparing the&nbsp;two&nbsp;versions to&nbsp;determine&nbsp;whether <code><var>version_added</var>&nbsp;&lt;&nbsp;<var>version_removed</var>.

Reply:
Ah, yeah, that’s a problem...  Nice catch!  Can you open up an issue for this (or a pull request, whichever you prefer) so we can deploy a fix ASAP?

Reply:
I’ve&nbsp;now opened&nbsp;<https://github.com/mdn/browser-compat-data/pull/5501> to&nbsp;fix&nbsp;this.